tests/base/src/test/java/org/wildfly/security/http/impl/AbstractBaseHttpTest.java

```diff
@@ -301,7 +330,10 @@ public boolean resumeRequest() {
         }
 
         public HttpScope getScope(Scope scope) {
-            return new HttpScope() {
+            if (scope.equals(Scope.SSL_SESSION))
```

Comment:
Very minor, but style wise, would be good to add the missing curly brackets in the if statement.